tests/controllers/UsersControllerTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace JoindinTest\Controller;
+
+
+class UsersControllerTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * Ensures that if the deleteUser method is called and no user_id is set,
+     * an exception is thrown
+     *
+     * @return void
+     *
+     * @test
+     * @expectedException        \Exception
+     * @expectedExceptionMessage You must be logged in to delete data
+     */
+    public function testDeleteUserWithNoUserIdThrowsException()
+    {
+        $request = new \Request([], ['REQUEST_URI' => "http://api.dev.joind.in/v2.1/users/3", 'REQUEST_METHOD' => 'DELETE']);
+
+        $usersController = new \UsersController();
+        $db = $this->getMockBuilder('\JoindinTest\Inc\mockPDO')->getMock();
```

Comment:
YOu might want to have a look at https://github.com/joindin/joindin-api/pull/340/files#diff-2e9dde7e960c27b3feffd135af2229adR108